LegMotorPositions left_front
LegMotorPositions right_front
LegMotorPositions left_middle
LegMotorPositions right_middle
LegMotorPositions left_rear
LegMotorPositions right_rear
